.github/workflows/full-maven-build.yml

```
@@ -6,9 +6,9 @@ name: Java CI with Maven
 env:
   JDK_VERSION: 17
   PROFILES: frankframework,codecoverage,ci
-  IS_GITHUB_DEPENDABOT: ${{ github.actor == 'dependabot[bot]' }}
-  # Don't allow credentials on forks, secrets are not available when using forks or GH bot
-  USE_PROPRIETARY: ${{ (github.event_name != 'pull_request' || github.event.pull_request.head.repo.owner.login == 'frankframework') && github.actor != 'dependabot[bot]' }}
+  IS_DEPENDENCY_BOT: ${{ github.actor == 'dependabot[bot]' || github.actor == 'renovate[bot]' }}
+  # Don't allow credentials on forks, secrets are not available when using forks or dependency bots
+  USE_PROPRIETARY: ${{ (github.event_name != 'pull_request' || github.event.pull_request.head.repo.owner.login == 'frankframework') && github.actor != 'dependabot[bot]' && github.actor != 'renovate[bot]' }}
```

```suggestion
  USE_PROPRIETARY: ${{ (github.event_name != 'pull_request' || github.event.pull_request.head.repo.owner.login == 'frankframework') && !IS_DEPENDENCY_BOT }}
```

Geen idee of dit kan hoor...

---

Nee het is helaas niet mogelijk om een eerder gedefinieerde environment variabele in een later gedefinieerde environment variabele te gebruiken..